### ISSUE TITLE
Matlab 7.3 support

### ISSUE THREAD

**Issue by RyanHope** [stewid/rmatio#6]
There seems to be a semi working mat73 branch which has issues loading a mat 7.3 file I have. Is this project still alive?

**Comment by stewid** [comment]
Yes the project is still alive, but unfortunately, I have not found time to add support for Matlab 7.3
